cpp/src/io/parquet/reader_impl_chunking.cu

```diff
@@ -1524,7 +1527,8 @@ void reader::impl::create_global_chunk_info()
       auto col = _input_columns[i];
       // look up metadata
       auto& col_meta = _metadata->get_column_metadata(rg.index, rg.source_index, col.schema_idx);
-      auto& schema   = _metadata->get_schema(col.schema_idx);
+      auto& schema   = _metadata->get_schema(
+        _metadata->map_schema_index(col.schema_idx, rg.source_index), rg.source_index);
```

Comment:
no need for is_schema_idx_mapped check here?

Reply:
No because at this point, we have gone through the mapping process and caught (excepted on) any mismatches. It was only needed at one place.